classes/profile.php

```diff
@@ -269,6 +281,11 @@ public static function save(\ExcimerLog $log, int $reason, int $created, float $
         if ($intrans) {
             $db2->dispose();
         }
+
+        // Clear the profile timings cache on insert/update of a profile.
+        $cache = \cache::make('tool_excimer', 'timings');
```

Comment:
isn't this too broad and we only want to delete a single cache item not the whole lot?